pandas/tseries/period.py

```diff
@@ -355,6 +355,36 @@ def freqstr(self):
         return self.freq
 
     def asfreq(self, freq=None, how='E'):
+        """
+        Convert the PeriodIndex to the specified frequency `freq`.
+
+        Parameters
+        ==========
+
+        freq: str; a frequency
```

Comment:
Some formatting notes: space between `freq` and `:`, and can you put "a frequency" on the next line?

The format has to follow this:

```
arg : type
    Explanation
```

rather strictly to render correctly